INSTALL.rst

```diff
@@ -82,6 +82,7 @@ To run outside of Docker you can use the Homebrew installation of Elasticsearch
 
     $ brew install elasticsearch
 
+Neither of these two methods is currently working for an M1 MacBook.
```

Comment:
So what are you doing currently?

Reply:
For now I'm just using Elasticsearch v7.1.1 installed with Homebrew.  I added a clarifying comment.  I guess we'll need to move to OpenSearch fairly soon, which might be easier to install on an M1 MacBook.